Packs/QRadar/Playbooks/playbook-QRadar_Generic.yml

```diff
@@ -1805,10 +1872,26 @@ tasks:
           - operator: uniq
       Username:
         complex:
-          root: Account
-          accessor: Username
+          root: QRadar.Username
           transformers:
+          - operator: SetIfEmpty
+            args:
+              applyIfEmpty: {}
+              defaultValue:
+                value:
+                  simple: Account.Username
```

Comment:
Shouldn't that be changed to QRadar.Username?

Reply:
Wanted to add a screenshot but it won't let me (browser?)
this is part of the `SetIfEmpty` transformer, but above it you can see QRadar.Username. So if QRadar Username doesn't exist, it will fallback to Account.Username (as it was before)